nopassword/forms.py

```diff
@@ -1,61 +1,133 @@
 # -*- coding: utf-8 -*-
 from django import forms
-from django.contrib.auth import authenticate
+from django.conf import settings
+from django.contrib.auth import authenticate, get_backends, get_user_model
+from django.contrib.sites.shortcuts import get_current_site
+from django.core.exceptions import ImproperlyConfigured
+from django.shortcuts import resolve_url
 from django.utils.translation import ugettext_lazy as _
 
-from .models import LoginCode
-from .utils import get_username_field
+from nopassword import models
 
 
-class AuthenticationForm(forms.Form):
-    """
-    Base class for authenticating users. Extend this to get a form that accepts
-    username logins.
-    """
-    username = forms.CharField(label=_("Username"), max_length=30)
-
+class LoginCodeRequestForm(forms.Form):
```

Comment:
Backward incompatible: Renamed form. `AuthenticationForm` is somehow misleading, as this form/view does NOT authenticate, the user won't have access after `form.save()`, it only sends the login code.